pyproject.toml

```diff
@@ -233,7 +238,7 @@ addopts = [
     "--strict-markers",
 ]
 testpaths = [
-    "docs/source/examples/",
+    "tests",
```

Comment:
If we were to add both `tests/` and `docs/source/examples/` here, could we skip specifying the path in `pytest --nbmake` above?

Reply:
Resolved in the meeting and we'll have to pass the path to the notebooks explicitly, it's already how it is like in our documentation: https://docs.pybamm.org/en/stable/source/user_guide/contributing.html#testing-the-example-notebooks